sql/moz-fx-data-shared-prod/glean_auto_events/auto_events_metadata_v1/query.sql

```diff
@@ -0,0 +1,24 @@
+WITH events AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    event,
+    JSON_VALUE(event_extra.id) AS element_id,
```

Comment:
I like using `LAX_STRING` instead as it's more forgiving of unexpected data. It's probably not a big deal here since there's filtering by event category/name.